src/interfaces/index.ts

```diff
@@ -1,6 +1,7 @@
 export type {
   AuthenticationResponse,
   AuthenticationResponseRaw,
+  OnRefreshResponse,
```

Comment:
is there a better name for this? onRefreshArgs? (it's the AuthenticationResponse minus the refresh token).